### PR TITLE
Add pre-curation checklist to Curator Guide

### PR DESCRIPTION
**Pull request by katherinesimeon**
- Added a Pre-Curation checklist to the curator guide   
- Added Cosima Meyer's blog post to curator artifacts  

Tagging @EmmanuelleRN for secondary review. Thanks!

### CODE REVIEW

File: content/rocur/guide/index.en.md [hunk 1]
```diff
@@ -156,7 +156,39 @@ Examples include:
...
+    
+To prepare for your curation week, we recommend the following steps to familiarize yourself with Tweetdeck and other processes that will be helpful when tweeting through your curation:    
+  
+### Basic: I only have an hour or less     
+  
+- [ ] **Send a Tweet from your personal account via Tweetdeck**. See more in our [FAQ](https://guide.rladies.org/rocur/faq/#1-how-do-i-start-my-first-tweet-as-wearerladies).   
+      - While logged into twitter, visit: https://tweetdeck.twitter.com/.  
+      - In the top left of the browser window, click the "Tweet" button, a menu should appear with a text box to enter your tweet. Be sure your personal account is selected if you are already connected to multiple accounts on Tweetdeck.  
+         
+- [ ] **Tweet an image with alternative text**.
```
Comment (maelle) [inline content/rocur/guide/index.en.md:171]: :heart:

File: content/rocur/guide/index.en.md [hunk 2]
```diff
@@ -156,7 +156,39 @@ Examples include:
...
+     
+### Intermediate: I have more time to prepare  
+  
+- [ ] **Schedule a Tweet from your personal account via Tweetdeck**.  
+       - In Tweetdeck, compose your tweet. Then, click "Schedule Tweet" and select the date and time you'd like the Tweet posted.
+       - Click Schedule Tweet at *date/time*.  
+          
+- [ ] **Practice creating a thread on your personal account via Tweetdeck**. See more in our [FAQ](https://guide.rladies.org/rocur/faq/#3-how-do-i-create-thread-via-tweet-deck).  
+     
+- [ ] **Create a list of topics you hope to tweet about during your curation week**. Many [previous](https://tolerancetoambiguity.com/my-week-with-we-are-r-ladies/) [curators](https://www.pipinghotdata.com/posts/2021-09-23-curating-for-wearerladies-on-twitter/) discuss how they planned out their content for the week (an example is below). Brainstorming content at any level of detail before your curation week starts will be helpful!
```
Comment (shannonpileggi) [inline content/rocur/guide/index.en.md:186]:
```suggestion
- [ ] **Create a list of topics you hope to tweet about during your curation week**. Previous curators have documented how they planned content for the week (see posts by [Divya Seernani](https://tolerancetoambiguity.com/my-week-with-we-are-r-ladies/) or [Shannon Pileggi](https://www.pipinghotdata.com/posts/2021-09-23-curating-for-wearerladies-on-twitter/)) . Brainstorming content at any level of detail before your curation week starts will be helpful!
```

Comment (shannonpileggi) [reply content/rocur/guide/index.en.md:186]: when creating links, i try to be explicit about what the links go to. it wouldn't have been clear that `previous` & `curators` went to two separate links.

**Review by shannonpileggi — APPROVED** [review]
Apologies for the delay!  I made one suggestion. This is a great checklist.

**Pull request merged**